container/container.go

```diff
@@ -1545,6 +1559,11 @@ func (b *ContainerWorker) CommandHandler(command worker.Command) bool {
 			serviceIdentity := cutil.FormOrgSpecUrl(cutil.NormalizeURL(ags[0].RunningWorkload.URL), ags[0].RunningWorkload.Org)
 			sVer := ags[0].RunningWorkload.Version
 
+			if err := b.GetSecretsManager().WriteServiceSecretsToFile(cmd.AgreementLaunchContext.Microservices, agreementId); err != nil {
```

Comment:
I think you need this in the ContainerConfigure path also.

Reply:
Moved it to ResourcesCreate() so it runs for both paths